autogluon_zeroshot/simulation/simulation_context.py

```diff
@@ -199,11 +200,12 @@ def load_pred(self, pred_pkl_path: Path, lazy_format: bool = False) -> dict:
         if lazy_format:
             # convert to lazy format if format not already available
             self.convert_lazy_format(pred_pkl_path=pred_pkl_path)
-        pred_path = str(pred_pkl_path.parent  / 'zeroshot_pred_per_task') if lazy_format else str(pred_pkl_path)
+        lazy_filename = str(pred_pkl_path).rsplit('.pkl', 1)[0]
+        pred_path = str(lazy_filename) if lazy_format else str(pred_pkl_path)
```

Comment:
This ensures that the lazy pred folder is unique to the pkl file in case multiple pkl files are in the same folder